cmd/initContainer/main.go

```diff
@@ -276,17 +285,62 @@ func removePolicyReport(client *client.Client, kind string) error {
 	}
 
 	// name of namespace policy report follows the name convention
-	// policyreport-ns-<namespace name>
+	// pr-ns-<namespace name>
 	for _, ns := range namespaces.Items {
-		reportName := fmt.Sprintf("pr-ns-%s", ns.GetName())
+		reportName := fmt.Sprintf("policyreport-ns-%s", ns.GetName())
```

Comment:
can we move this to a common function (getReportName) and reuse from all places so its easier to maintain?